BUILD.bazel

```diff
@@ -1076,6 +1076,20 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "cluster_resource_scheduler_2_test",
```

Comment:
To be clear:
Because `gcs_resource_scheduler` is refactored into `cluster_resource_scheduler`, so `gcs_resource_scheduler_test` is also refactored into `cluster_resource_scheduler_2_test`.